### DISCUSSION TITLE
In Termux host environment, I have twice tried to install either miniforge3 or mamba, but have had errors with the installation scripts

### DISCUSSION THREAD

**Discussion by b9Joker108** [conda-forge/miniforge#582] [Q&A]
Hello

In the Termux host environment, I have twice endeavoured to install either miniforge3 or mamba, but have had errors with the installation scripts. I have succesfully installed miniforge3 (conda), in a chroot, with a guest proot-distro-alias of Debian GNU/Linux. In this Debian envoronment, Conda has been a revelation. Is it possible somehow, to install Conda within the unrooted Termux host?

Any assistance greatly appreciated.

Thanking you in anticipation

**Comment by holonen** [comment]
the file _conda in the directory miniforge3 is not found 
despite:

pi@raspberrypi:~$ cd miniforge3/
pi@raspberrypi:~/miniforge3$ ls
_conda  install_tmp

and:

pi@raspberrypi:~/miniforge3$ file /home/pi/miniforge3/_conda
/home/pi/miniforge3/_conda: ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, stripped

permissions for _conda seems ok

Tried installation with sudo also. Same problem.

**Reply by b9Joker108** [reply]
Thank you so much for helping.
